assessment.md

```diff
@@ -13,81 +13,512 @@
+  * I = A few comments in the code that are not detailed
+  * A = Documentation has a bit of detail for every main part of the program
+  * G = Documentation is detailed and visible for most of the software within the
+  source code
+  * E = Explains in great detail how to use and operate the computer software
+  within the source code
 * Programming styles
+  * N = Programming style is not set and always changing around during the program
+  * I = Not a clear programming style but has some implementation of general styles
+  * A = Some clarity of a programming style with multiple styles implemented
+  * G = Programming style is implemented well with a few variances throughout
+  * E = One distinct style of programming that is clear and easy to understand
 * Managing software complexity
+  * N = The code does not run properly, files are not in the right place, or some
+  other part of the project has been completely broken
+  * I = Little effort is put forth to keep code clean or organized, some code may
+  not work properly
+  * A = Basic organization is achieved, no major parts of the projects are
+  affected by any lax in management
+  * G = The project is logically and effectively organized with no issues in the
+  code resulting from poor organization
+  * E = Everything is exceedingly well organized, with no excess or lack of files,
+  folders, branches, forks, etc.
 * Mitigating software risk
+  * N = No procedure is followed, the master branch is broken, or files are corrupted
+  * I = The engineer avoids causing damage to the work of others, but may not produce
+  any safely compatible source code
+  * A = Major issues are avoided and a few small complications may occur, none that
+  ruin the flow of work for the team
+  * G = Almost no problems occur while working with code or branches, does not slow
+  down productivity for self or others
+  * E = All interactions with new and existing code are free from complications
+  or other issues, everything works smoothly and completely free of errors
 
 ## Professional Skills
 
 ### Individual
 
 * Continuous learning about
   * Python software development
-  * Project management with GitHub
+    * N = A lack of new knowledge and/or contributions
+        to any project
+    * I = Incomplete knowledge or minuscule contributions
+        to projects
+    * A = An average amount of knowledge and contributions
+        to each project
+    * G = An above average amount of knowledge and contributions
+        to each project
+    * E = Going above and beyond the expectations and contributions
+        of others
+  * Project management with GitHub, specifically the Flow Model
+    * N = A complete disregard for the model, especially multiple
+        instances of overriding the master branch
+    * I = Occasional disregard for the model including overriding
+        the master branch
+    * A = Almost always adhering to model, but never overriding
```

Comment:
What is "overriding the master branch"? Correct terminology should be used.

Reply:
This was fixed by my pull request [#15](https://github.com/huangs1/assessment-professional-skills/pull/15)